dashboard/test/unit/lib/store/table.test.js

```diff
@@ -0,0 +1,47 @@
+const expect = require('chai').expect;
+
+// unit under test
+const Table = require('../../../../lib/store/table');
+
+describe('Table class', () => {
+    let table, mockData;
+
+    before('setup table mock', () => {
+        table = new Table({});
+        mockData = {
+            id: 'abc',
+            arg: 'arg',
+        };
+    });
+
+    it('should initialize the cache correctly', async () => {
+        const cache = await table.find();
+        expect(cache).to.deep.equal({});
+    });
+
+    it('should insert new item to cache', async () => {
+        const item = await table.insert(mockData.id, mockData);
+        expect(item).to.be.undefined;
+    });
+
+    it('should fetch item from cache', async () => {
+        const item = await table.findOne(mockData.id);
+        expect(item).to.deep.equal(mockData);
+    });
```

Comment:
Should also test "sad cases" for the methods, like:
1. Trying to fetch invalid ID.
2. Trying to fetch with ID as an array.
3. Trying to remove and ID that does not exist.
4. etc.

Reply:
Done!